connector/spanmetricsconnector/connector.go

```diff
@@ -411,6 +427,7 @@ func (p *connectorImp) aggregateMetrics(traces ptrace.Traces) {
 				s, limitReached := sums.GetOrCreate(key, attributesFun, startTimestamp)
 				if !limitReached && p.config.Exemplars.Enabled && !span.TraceID().IsEmpty() {
 					s.AddExemplar(span.TraceID(), span.SpanID(), duration)
+					p.markSpanAsExemplar(span)
```

Comment:
Please notice that:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/a6f867bca566b9b27a8f350b019cb26469aa59dc/connector/spanmetricsconnector/internal/metrics/metrics.go#L272-L282

Please consider this case:  If the maxExemplarCount is reached, the span can't be added into exemplars